drivers/modem/Kconfig.gsm

```diff
@@ -78,6 +78,20 @@ config MODEM_GSM_MANUAL_MCCMNO
 	  the network provider and may need to be changed if auto is not
 	  selected.
 
+config MODEM_GSM_RSSI_WORK_PERIOD
+	int "Configure RSSI WORK polling period"
+	default 30
+	help
+	  This settings is used to configure the period of RSSI polling
+
+config MODEM_GSM_ENABLE_CESQ_RSSI
+	bool "Enable +CESQ RSSI measurement"
+	default n
```

Comment:
Options are not set by default so this can be removed.

Reply:
addressed, thanks